Homework_week_5_6/Homework_week_5_6.md

```diff
@@ -0,0 +1,109 @@
+**Question 1**
+
+· Product backlog refinement- breaking down and refining the product backlog items to smaller items
+
+· Sprint planning – starts the sprint by planning for the work to be done in the sprint. The whole team agrees to complete a set of product backlog items.
+
+· Daily scrum- a daily meeting to find out the daily progress of the team. Each person answers what they are working on today, what they did yesterday and what impediments are in the way. 
+
+· Sprint review- A meeting held at the end of a sprint which the team shows the team has done during the sprint. It is like a demo of new features added in the product
+
+· Sprint retrospective- the sprint team looks at what went well, what didn’t go well and what could be improved for future sprints.
+
+
+SCRUM ROLES
+
+· ScrumMaster- ensures that the scrum framework is followed. They facilitate that team members follow scrum practices. They arrange the scrum meetings.
+
+· Product Owner- They set out the items on the product backlog.
+
+They order and prioritize the items on the product backlog. They are responsible for knowing and managing stakeholders needs.
+
+· Development Team- Team of professionals that do the work to deliver a potentially viable product at the end of each Sprint.
+
+Question 2
```

Comment:
6/7 This could be explained in more detail up to technical tasks. i.e. who will be responsible for creating the API, designing the database schema, etc.